ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -134,13 +134,16 @@ monitoring_pod="$( oc --insecure-skip-tls-verify get pods -n openshift-monitorin
 queue ${ARTIFACT_DIR}/metrics/prometheus.tar.gz oc --insecure-skip-tls-verify exec -n openshift-monitoring "${monitoring_pod}" -- tar cvzf - -C /prometheus .
 
 cat >> ${SHARED_DIR}/custom-links.txt << EOF
+<p>
 <script>
 let a = document.createElement('a');
 a.href="https://promecieus.dptools.openshift.org/?search="+document.referrer;
 a.innerHTML="PromeCIeus";
 a.target="_blank";
 document.getElementById("wrapper").append(a);
 </script>
+  - PromeCIeus runs a monitoring stack based on the metrics dump from a CI run.  Commonly used for inspecting alert failures.
```

Comment:
This didn't render like the others.  See the "custom-link-tools.html" on the https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20886/rehearse-20886-pull-ci-openshift-installer-release-4.9-e2e-gcp-upgrade/1422911631804862464 job run.

Any html advice for me?